met-web/src/components/engagement/new/create/index.tsx

```diff
@@ -85,6 +88,32 @@ const EngagementCreationWizard = () => {
         formState: { errors, isDirty, isValid, isSubmitting },
     } = engagementCreationForm;
 
+    const handleLanguageColumn = (languages: Language[], minIndex: number, maxIndex: number) => {
+        return languages.map((language: Language, index: number) => {
+            return (
+                minIndex <= index &&
+                maxIndex >= index && (
+                    <>
+                        <span>
+                            <Checkbox
+                                defaultChecked={'English' === language.name && true}
```

Comment:
```suggestion
                                defaultChecked={Boolean('English' === language.name)}
```
Are these `&& true` statements doing anything?

Reply:
Yes they work, but I don't mind doing it the other way